addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -41,6 +41,8 @@ odoo.define('website.s_website_form', function (require) {
         xmlDependencies: ['/website/static/src/xml/website_form.xml'],
         events: {
             'click .s_website_form_send, .o_website_form_send': 'send', // !compatibility
+            'change input[type=file]': '_onFileChange',
+            'click input[class="fake-button mr-1"]': '_onFakeButtonClick',
```

Comment:
This will need to be maintained in case `mr-1` changes to something else.
Maybe you can use `input.fake-button` (if the hyphen prevents this from working, try renaming the class with underscore instead - maybe even `o_fake_button`).